src/components/standalone/users_objects/CreateOrEditHostSetDrawer.vue

```diff
@@ -105,7 +106,18 @@ watch(
   }
 )
 
+// compute redirectItemsName the name of the portforward rule using this object
+watch(
+  () => props.currentHostSet?.matches,
+  async () => {
+    if (props.currentHostSet?.matches) {
+      redirectItemsName.value = await getMatchedItemsName(props.currentHostSet.matches)
+    }
+  }
+)
+
 function closeDrawer() {
+  emit('reloadData')
```

Comment:
Is this needed?

Reply:
yes else the UI keeps the entry you tried to validate when you close the drawer until you reload the whole page

Reply:
Understood. This is a pre-existing glitch, to solve it let's `import { cloneDeep } from 'lodash-es'`  and change line 106 to:
```js
records.value = cloneDeep(props.currentHostSet.ipaddr)
```